components/validator/guardian.go

```diff
@@ -176,12 +190,21 @@ func (g *Guardian) processOutputValidation(ctx context.Context, event *bindings.
 	Loop:
 		select {
 		case <-ticker.C:
-			cCtx, cCancel := context.WithTimeout(ctx, g.cfg.NetworkTimeout)
-			callOpts := utils.NewCallOptsWithSender(cCtx, g.cfg.TxManager.From())
-			isConfirmed, err := g.securityCouncilContract.IsConfirmed(callOpts, event.TransactionId)
-			cCancel()
+			outputIndex := new(big.Int).Sub(event.L2BlockNumber, g.submissionInterval)
```

Comment:
I think that the outputIndex should be obtained via `l2ooContract.getL2OutputIndexAfter()`.